### DISCUSSION TITLE
modalita compatina non riconosce i metri del compattatotore

### DISCUSSION THREAD

**Discussion by marcolino04** [Courseplay/Courseplay_FS22#2257] [General]
ciao come da titolo la modalita compatina non riconocsce piu i metri di latoro dei atrezzi compattanti, saluti grazie

**Comment by Tensuko** [comment]
Can you tell me what exact tool?
Screenshot?
Google Translator doesn't know that exactly.

**Comment by marcolino04** [comment]
![Screenshot (30)](https://user-images.githubusercontent.com/104140737/212963532-b2cb6fdb-3a80-490e-a703-3756bc02d034.png)
qyesto

**Reply by Tensuko** [reply]
Should be clicking a few times on the starting point untill the Bunker mode appear.
This is due to roller also exist for fieldwork.

**Reply by marcolino04** [reply]
Capito, grazie. Quando si potrà usare una pala gommata per svuotare. Il bunker ?ciao buona serata

**Reply by Tensuko** [reply]
Not yet, but is planed
